src/views/docs/en/reference/app.arc/static.md

```diff
@@ -99,6 +98,8 @@ static:
 </div>
 </arc-viewer>
 
+> 🔖 The [Frontend Static assets guide](/docs/en/guides/frontend/static-assets) has more information on how to use static assets in your Architect project.
```

Comment:
Should we have a space between the bookmark emoji and `The`? Just thinking a screen reader may really mangle that pronunciation.

Reply:
There is a space character. maybe GitHub doesn't kern it well in their code font. Looks like this with the space highlighted
<img width="483" alt="Screen Shot 2021-09-23 at 10 30 15 AM" src="https://user-images.githubusercontent.com/15697/134547205-cc56b013-120d-41d4-bbe9-3ff41f84e627.png">

Maybe a double space?